### PR TITLE
Build Farm 2 Initial review from RHMAP doc team

### PR DESCRIPTION
**Pull request by pmacko1**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RUN useradd -d /home/aerogear -G rvm aerogear
+WORKDIR /home/aerogear
+USER aerogear
+RUN source /etc/profile.d/rvm.sh
+RUN git clone https://github.com/aerogear/aerogear.org aerogear
+
+WORKDIR /home/aerogear/aerogear
+
+## Setup Project
+  RUN  /bin/bash -l -c "rvm install ruby 2.1.2"
```
Comment (wei-lee) [inline docker/Dockerfile]: indentation

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+USER root
+RUN dnf install -y libcurl-devel libcurl
+USER aerogear
+ 
+ RUN  /bin/bash -l -c "bundle install --path vendor"
+
+## Run
+ENTRYPOINT /bin/bash -l -c "bundle install --path vendor;bundle exec jekyll serve --host=0.0.0.0 --watch"
+
+EXPOSE 4000
```
Comment (wei-lee) [inline docker/Dockerfile]: nice 👍

**Comment by wei-lee** [comment]
let's get this merged

**Comment by pmacko1** [comment]
do i have to merge?

**Pull request merged**